ouroboros-consensus/src/Ouroboros/Consensus/Storage/ChainDB/Impl/Types.hs

```diff
@@ -552,7 +570,7 @@ data TraceAddBlockEvent blk
     -- as the new chain.
 
   | SwitchedToAFork
-      (RealPoint blk)
```

Comment:
Keep the `RealPoint`, it corresponds to the block for which we ran chain selection, which is not necessarily the new tip. Turn it into the `newTipTrigger` field of `NewTipInfo`. Now the explanation can go there.